fflib/v1/lexer.go

```diff
@@ -471,6 +470,7 @@ func (ffl *FFLexer) Scan() FFTok {
 		default:
 			tok = FFTok_error
 			ffl.Error = FFErr_invalid_char
+			goto lexed
```

Comment:
This one fixes the `NaN` bug (issue #241). Its a complex one line change and I can't fully see the consequences but all tests pass after the change.